### ISSUE TITLE
[SB8] "Author image" is missing a title and alt text

### ISSUE THREAD

**Issue by theadamparker** [undp/design-system#1364]
The markup has changed. See comparison screenshot:

![Screenshot 2024-09-12 at 5 12 31 PM](https://github.com/user-attachments/assets/a8e67564-2b59-400b-a90a-c81068c8d986)

https://develop.design-system.pages.dev/?path=/story/foundation-images-author-image--author-image

**Comment by CalebLuster** [comment]
fixed

**Comment by Jura** [comment]
Verified
